docs/en/dev_setup/dev_env_mac.md

```diff
@@ -72,9 +55,67 @@ To setup the environment to be able to build for Pixhawk/NuttX hardware (and ins
    sudo -H python3 -m pip install --user pyserial empty toml numpy pandas jinja2 pyyaml pyros-genmsg packaging kconfiglib future jsonschema
    ```
 
-## Gazebo Classic Simulation
+## Gazebo Simulation
+
+To setup the environment for [Gazebo](../sim_gazebo_gz/index.md) simulation with Gazebo Harmonic:
+
+1. Install Gazebo Harmonic and required dependencies:
+
+   ```sh
+   brew install gz-harmonic
+   brew install qt@5 gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav
+   ```
+
+1. Configure Qt5 and library paths by adding the following to your `~/.zshrc` file:
+
+   ```sh
+   # Qt5 configuration
+   export Qt5_DIR=/opt/homebrew/opt/qt@5/lib/cmake/Qt5
+   export PATH="/opt/homebrew/opt/qt@5/bin:$PATH"
+   export CMAKE_PREFIX_PATH="/opt/homebrew/opt/qt@5:$CMAKE_PREFIX_PATH"
+
+   # Library paths for GStreamer and other dependencies
+   export PKG_CONFIG_PATH="/opt/homebrew/lib/pkgconfig:$PKG_CONFIG_PATH"
+   export LIBRARY_PATH="/opt/homebrew/lib:$LIBRARY_PATH"
+   export DYLD_LIBRARY_PATH="/opt/homebrew/lib:$DYLD_LIBRARY_PATH"
+   ```
+
+1. Link Qt5 and reload your shell configuration:
```

Comment:
I recall that we had this discussion before. Although this can address a few initial issues, it's essentially a "magic" solution that isn't fully explained, and we shouldn't recommend it without also explaining its necessity. Let's find a way to be more explicit about it.